pkg/apis/autopilot/v1alpha1/actionapproval.go

```diff
@@ -14,6 +14,8 @@ const (
 	ApprovalStateApproved ActionApprovalState = "approved"
 	// ApprovalStateDeclined  means the action has been declined
 	ApprovalStateDeclined ActionApprovalState = "declined"
+	// ApprovalStateCanceled  means the action has been canceled
```

Comment:
nit: change comment to `action approval has been canceled`

Reply:
done